src/models/dtos/multi-region-config.dto.ts

```diff
@@ -0,0 +1,38 @@
+import { IAuthConfigOptions, SearchEngineType } from './config.dto';
+
+export type Region = 'singapore' | 'indonesia' | 'australia' | 'thailand';
```

Comment:
@sguhaprasad Could you please remove the fixed type and add it as a generic injected from PAH functional service, remember this library is not just for PAH...the idea is to put it as a generic injected the 
```suggestion
IMultiregionalOpensearchConfig<Region>
```
Then declare the type in the FS and add it as a generic. 
It will allow to keep this library generic.

Reply:
sure, I will make this change